src/compat/db_macro.c

```diff
@@ -6867,3 +6867,35 @@ db_get_json_document (const DB_VALUE * value)
 
   return value->data.json.document;
 }
+
+int
+db_get_deep_copy_of_json (const DB_JSON * src, DB_JSON * dst)
+{
+  CHECK_1ARG_NULL (src);
+
+  char *raw_json_body = NULL, *raw_schema_body = NULL;
+  JSON_DOC *doc_copy = NULL;
+
+  raw_json_body = (char *) db_private_strdup (NULL, src->json_body);
```

Comment:
db_private_strdup returns char *. please remove all unnecessary casts (for all db_private_strdup references).

Reply:
done